android/ANDROID.md

```diff
@@ -0,0 +1,124 @@
+
+API documentation: https://awslabs.github.io/aws-crt-java/
+
+This SDK is built on the AWS Common Runtime, a collection of libraries
+([aws-c-common](https://github.com/awslabs/aws-c-common),
+[aws-c-io](https://github.com/awslabs/aws-c-io),
+[aws-c-mqtt](https://github.com/awslabs/aws-c-mqtt),
+[aws-c-http](https://github.com/awslabs/aws-c-http),
+[aws-c-cal](https://github.com/awslabs/aws-c-cal),
+[aws-c-auth](https://github.com/awslabs/aws-c-auth),
+[s2n](https://github.com/awslabs/s2n)...) written in C to be
+cross-platform, high-performance, secure, and reliable.
+
+*__Jump To:__*
+
+* [Installation](#installation)
+  * [Minimum requirements](#minimum-requirements)
+  * [Build and install CRT from source](#build-and-install-crt-from-source)
+* [Consuming AWS CRT Android](#consuming-aws-crt-android)
+  * [Consuming from Maven](#consuming-from-maven)
+  * [Consuming from locally installed](#consuming-from-locally-installed)
+
+## Installation
+
+### Minimum requirements
+* Java 11+ ([Download and Install Java](https://www.java.com/en/download/help/download_options.html))
+  * [Set JAVA_HOME](#set-java_home)
+* Gradle 7.4.2 ([Download and Install Gradle](https://gradle.org/install/))
+* Android SDK 24 ([Doanload SDK Manager](https://developer.android.com/tools/releases/platform-tools#downloads))
+  * [Set ANDROID_HOME](#set-android_home)
+* Android NDK ([Download and install Android NDK](https://developer.android.com/ndk/downloads))
+
+### Build and install CRT from source
+Supports API 24 or newer.
+
+``` sh
+# Create a workspace directory to hold all the SDK files
+mkdir sdk-workspace
+cd sdk-workspace
+# Clone the CRT repository
+# (Use the latest version of the CRT here instead of "v0.27.6)
+git clone --branch v0.27.6 --recurse-submodules https://github.com/awslabs/aws-crt-java.git
+# Compile and install the CRT for Android
+./gradlew :android:crt:build
+# Install CRT locally
+./gradlew :android:crt:publishToMavenLocal
+```
+
+## Consuming AWS CRT Android
+
+### Consuming from Maven
+(#consuming-from-locally-installed)
+Consuming this CRT via Maven is the preferred method of consuming it and using it within your application. To consume
+AWS CRT Android in your application, add the following to your `build.gradle` repositories and dependencies:
+
+``` groovy
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    api 'software.amazon.awssdk.crt:aws-crt-android:0.27.6'
+}
+```
+Replace `0.27.6` in `software.amazon.awssdk.crt:aws-crt-android:0.27.6` with the latest release version of the CRT library.
+Look up the latest SDK version here: https://github.com/awslabs/aws-crt-java/releases
+
+### Consuming from locally installed
+You may also consume AWS CRT Android in your application using a locally installed version by adding the
+following to your `build.gradle` repositories and depenencies:
+``` groovy
+repositories {
+    mavenLocal()
+}
+
+dependencies {
+    api 'software.amazon.awssdk.crt:aws-crt-android:0.27.6'
```

Comment:
As the CRT local build will use `1.0.0-SNAPSHOT` by default, it might make more intuitive to have `1.0.0-SNAPSHOT` in the sample code here.

Reply:
I think we removed the building of 1.0.0-SNAPSHOT to work with CD. I'll test now and change it if the local install instructions above produce the 1.0.0-SNAPSHOT version.

Reply:
Confirmed that it's publishing to maven local under the current version. We can force it to publish as 1.0.0-SNAPSHOT but that may just further confuse the matter.